ostree: 2024.10 -> 2025.2

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://github.com/ostreedev/ostree/releases.


meta.description for ostree is: Git for operating system binaries

meta.homepage for ostree is: https://ostree.readthedocs.io/en/latest/

Previous: https://github.com/NixOS/nixpkgs/pull/293303
Previous: https://github.com/NixOS/nixpkgs/pull/366997

###### Updates performed
- Version update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/ostreedev/ostree/releases/tag/v2024.5)

- [Compare changes on GitHub](https://github.com/ostreedev/ostree/compare/v2024.4...v2024.5)

###### Impact

<b>Checks done</b>

---

- built on NixOS
- Warning: a test defined in `passthru.tests` did not pass
- found 2024.5 with grep in /nix/store/5jbriz2md9199j16p1p5cakczihnkypk-ostree-2024.5
- found 2024.5 in filename of file in /nix/store/5jbriz2md9199j16p1p5cakczihnkypk-ostree-2024.5

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
33 total rebuild path(s)

33 package rebuild(s)

First fifty rebuilds by attrpath
cosmic-store
flatpak
flatpak-builder
fractal
gnome-builder
gnome.gnome-software
haskellPackages.gi-ostree
haskellPackages.ostree-pin
kdePackages.discover
kdePackages.flatpak-kcm
libsForQt5.discover
libsForQt5.flatpak-kcm
malcontent-ui
monitor
ostree
ostree-rs-ext
pantheon.appcenter
pantheon.elementary-greeter
pantheon.elementary-onboarding
pantheon.sideload
pantheon.switchboard-plug-applications
pantheon.switchboard-with-plugs
pantheon.wingpanel-applications-menu
pantheon.wingpanel-with-indicators
plasma5Packages.discover
plasma5Packages.flatpak-kcm
rpm-ostree
vala-language-server
xdg-desktop-portal
xdg-desktop-portal-gnome
xdg-desktop-portal-gtk
xdg-desktop-portal-shana
xdg-desktop-portal-xapp
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/5jbriz2md9199j16p1p5cakczihnkypk-ostree-2024.5 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A ostree https://github.com/r-ryantm/nixpkgs/archive/b8e74d827a580352710dc4f4af6b1debf9c91e4b.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/b8e74d827a580352710dc4f4af6b1debf9c91e4b#ostree
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/5jbriz2md9199j16p1p5cakczihnkypk-ostree-2024.5
ls -la /nix/store/5jbriz2md9199j16p1p5cakczihnkypk-ostree-2024.5/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review --extra-nixpkgs-config '{ allowInsecurePredicate = x: true; }'` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages failed to build:</summary>
  <ul>
    <li>gnome-builder</li>
    <li>gnome-builder.devdoc</li>
    <li>vala-language-server</li>
  </ul>
</details>
<details>
  <summary>48 packages built:</summary>
  <ul>
    <li>cosmic-store</li>
    <li>flatpak</li>
    <li>flatpak-builder</li>
    <li>flatpak-builder.doc</li>
    <li>flatpak-builder.installedTests</li>
    <li>flatpak-builder.man</li>
    <li>flatpak.dev</li>
    <li>flatpak.devdoc</li>
    <li>flatpak.doc</li>
    <li>flatpak.installedTests</li>
    <li>flatpak.man</li>
    <li>fractal</li>
    <li>gnome.gnome-software</li>
    <li>kdePackages.discover</li>
    <li>kdePackages.discover.debug</li>
    <li>kdePackages.discover.dev</li>
    <li>kdePackages.flatpak-kcm</li>
    <li>kdePackages.flatpak-kcm.debug</li>
    <li>kdePackages.flatpak-kcm.dev</li>
    <li>libsForQt5.discover</li>
    <li>libsForQt5.flatpak-kcm</li>
    <li>malcontent-ui</li>
    <li>malcontent-ui.dev</li>
    <li>malcontent-ui.lib</li>
    <li>monitor</li>
    <li>ostree</li>
    <li>ostree-rs-ext</li>
    <li>ostree.dev</li>
    <li>ostree.installedTests</li>
    <li>ostree.man</li>
    <li>pantheon.appcenter</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-onboarding</li>
    <li>pantheon.sideload</li>
    <li>pantheon.switchboard-plug-applications</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>rpm-ostree</li>
    <li>rpm-ostree.dev</li>
    <li>rpm-ostree.devdoc</li>
    <li>rpm-ostree.man</li>
    <li>xdg-desktop-portal</li>
    <li>xdg-desktop-portal-gnome</li>
    <li>xdg-desktop-portal-gtk</li>
    <li>xdg-desktop-portal-shana</li>
    <li>xdg-desktop-portal-xapp</li>
    <li>xdg-desktop-portal.installedTests</li>
  </ul>
</details>

---

###### Maintainer pings

cc @copumpkin for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

jtojnar committed:
ostree: Re-add patches that were incorrectly removed

Partially reverts 1a56b3515b8434f91af104e987b05227a0b28eff

Also make the patch comments clearly mention that they are for *installed* tests.

alyssais commented:
@OfBorg build pkgsMusl.ostree

jtojnar commented:
Looks like the installed test here are another victim of GLib bindings no longer being shipped with gobject-introspection (https://github.com/NixOS/nixpkgs/pull/306080#issuecomment-2081520383)

r-ryantm commented:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://github.com/ostreedev/ostree/releases.


meta.description for ostree is: Git for operating system binaries

meta.homepage for ostree is: https://ostreedev.github.io/ostree/




###### Updates performed
- Version update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/ostreedev/ostree/releases/tag/v2024.9)

- [Compare changes on GitHub](https://github.com/ostreedev/ostree/compare/v2024.8...v2024.9)

###### Impact

<b>Checks done</b>

---

- built on NixOS

> [!CAUTION]
> A test defined in `passthru.tests` did not pass.
- found 2024.9 with grep in /nix/store/ivygpdq3m4lbmz2wc513ab149j1bcixw-ostree-2024.9
- found 2024.9 in filename of file in /nix/store/ivygpdq3m4lbmz2wc513ab149j1bcixw-ostree-2024.9

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
37 total rebuild path(s)

37 package rebuild(s)

First fifty rebuilds by attrpath

bootc
cosmic-store
flatpak
flatpak-builder
fractal
gnome-builder
gnome-software
gnome.gnome-software
haskellPackages.gi-ostree
haskellPackages.ostree-pin
kdePackages.discover
kdePackages.flatpak-kcm
libsForQt5.discover
libsForQt5.flatpak-kcm
malcontent-ui
monitor
ostree
ostree-rs-ext
pantheon.appcenter
pantheon.elementary-greeter
pantheon.elementary-onboarding
pantheon.sideload
pantheon.switchboard-plug-applications
pantheon.switchboard-with-plugs
pantheon.wingpanel-applications-menu
pantheon.wingpanel-with-indicators
plasma5Packages.discover
plasma5Packages.flatpak-kcm
rpm-ostree
streamcontroller
vala-language-server
xdg-desktop-portal
xdg-desktop-portal-gnome
xdg-desktop-portal-gtk
xdg-desktop-portal-shana
xdg-desktop-portal-xapp
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/ivygpdq3m4lbmz2wc513ab149j1bcixw-ostree-2024.9 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A ostree https://github.com/r-ryantm/nixpkgs/archive/1495e07d4451230927615c9b9a87527b977563a6.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/1495e07d4451230927615c9b9a87527b977563a6#ostree
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/ivygpdq3m4lbmz2wc513ab149j1bcixw-ostree-2024.9
ls -la /nix/store/ivygpdq3m4lbmz2wc513ab149j1bcixw-ostree-2024.9/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review`

---
### `x86_64-linux`
<details>
  <summary>:x: 8 packages failed to build:</summary>
  <ul>
    <li>fractal</li>
    <li>streamcontroller</li>
    <li>xdg-desktop-portal</li>
    <li>xdg-desktop-portal-gnome</li>
    <li>xdg-desktop-portal-gtk</li>
    <li>xdg-desktop-portal-shana</li>
    <li>xdg-desktop-portal-xapp</li>
    <li>xdg-desktop-portal.installedTests</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 48 packages built:</summary>
  <ul>
    <li>bootc</li>
    <li>cosmic-store</li>
    <li>flatpak</li>
    <li>flatpak-builder</li>
    <li>flatpak-builder.doc</li>
    <li>flatpak-builder.installedTests</li>
    <li>flatpak-builder.man</li>
    <li>flatpak.debug</li>
    <li>flatpak.dev</li>
    <li>flatpak.devdoc</li>
    <li>flatpak.doc</li>
    <li>flatpak.installedTests</li>
    <li>flatpak.man</li>
    <li>gnome-builder</li>
    <li>gnome-builder.devdoc</li>
    <li>gnome-software</li>
    <li>kdePackages.discover</li>
    <li>kdePackages.discover.debug</li>
    <li>kdePackages.discover.dev</li>
    <li>kdePackages.discover.devtools</li>
    <li>kdePackages.flatpak-kcm</li>
    <li>kdePackages.flatpak-kcm.debug</li>
    <li>kdePackages.flatpak-kcm.dev</li>
    <li>kdePackages.flatpak-kcm.devtools</li>
    <li>libsForQt5.discover</li>
    <li>libsForQt5.flatpak-kcm</li>
    <li>malcontent-ui</li>
    <li>malcontent-ui.dev</li>
    <li>malcontent-ui.lib</li>
    <li>monitor</li>
    <li>ostree</li>
    <li>ostree-rs-ext</li>
    <li>ostree.dev</li>
    <li>ostree.installedTests</li>
    <li>ostree.man</li>
    <li>pantheon.appcenter</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-onboarding</li>
    <li>pantheon.sideload</li>
    <li>pantheon.switchboard-plug-applications</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>rpm-ostree</li>
    <li>rpm-ostree.dev</li>
    <li>rpm-ostree.devdoc</li>
    <li>rpm-ostree.man</li>
    <li>vala-language-server</li>
  </ul>
</details>

---

###### Maintainer pings

cc @copumpkin for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

jtojnar committed:
ostree: 2024.10 → 2025.2

https://github.com/ostreedev/ostree/releases/tag/v2025.1
https://github.com/ostreedev/ostree/releases/tag/v2025.2

r-ryantm commented:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://github.com/ostreedev/ostree/releases.


meta.description for ostree is: Git for operating system binaries

meta.homepage for ostree is: https://ostreedev.github.io/ostree/




###### Updates performed
- Version update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/ostreedev/ostree/releases/tag/v2025.1)

- [Compare changes on GitHub](https://github.com/ostreedev/ostree/compare/v2024.10...v2025.1)

###### Impact

<b>Checks done</b>

---

- built on NixOS

> [!CAUTION]
> A test defined in `passthru.tests` did not pass.
- found 2025.1 with grep in /nix/store/xn1kas1lcynx3n1snzqzg9ifgji7kwzs-ostree-2025.1
- found 2025.1 in filename of file in /nix/store/xn1kas1lcynx3n1snzqzg9ifgji7kwzs-ostree-2025.1

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
37 total rebuild path(s)

37 package rebuild(s)

First fifty rebuilds by attrpath

bootc
cosmic-store
flatpak
flatpak-builder
fractal
gnome-builder
gnome-software
haskellPackages.gi-ostree
haskellPackages.ostree-pin
kdePackages.discover
kdePackages.flatpak-kcm
libsForQt5.discover
libsForQt5.flatpak-kcm
malcontent-ui
monitor
ostree
ostree-full
ostree-rs-ext
pantheon.appcenter
pantheon.elementary-greeter
pantheon.elementary-onboarding
pantheon.sideload
pantheon.switchboard-plug-applications
pantheon.switchboard-with-plugs
pantheon.wingpanel-applications-menu
pantheon.wingpanel-with-indicators
plasma5Packages.discover
plasma5Packages.flatpak-kcm
rpm-ostree
streamcontroller
vala-language-server
xdg-desktop-portal
xdg-desktop-portal-gnome
xdg-desktop-portal-gtk
xdg-desktop-portal-shana
xdg-desktop-portal-xapp
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from the cache**:
```
nix-store -r /nix/store/xn1kas1lcynx3n1snzqzg9ifgji7kwzs-ostree-2025.1 \
  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update-cache.nix-community.org/' \
  --option trusted-public-keys '
  nixpkgs-update-cache.nix-community.org-1:U8d6wiQecHUPJFSqHN9GSSmNkmdiFW7GW7WNAnHW0SM=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The nixpkgs-update cache is only trusted for this store-path realization.)
For the cached download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A ostree https://github.com/r-ryantm/nixpkgs/archive/3ba33b6dcfea237bec9f1dc1358112cbe64f7cd1.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/3ba33b6dcfea237bec9f1dc1358112cbe64f7cd1#ostree
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/xn1kas1lcynx3n1snzqzg9ifgji7kwzs-ostree-2025.1
ls -la /nix/store/xn1kas1lcynx3n1snzqzg9ifgji7kwzs-ostree-2025.1/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>libsForQt5.discover</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 59 packages built:</summary>
  <ul>
    <li>bootc</li>
    <li>cosmic-store</li>
    <li>flatpak</li>
    <li>flatpak-builder</li>
    <li>flatpak-builder.doc</li>
    <li>flatpak-builder.installedTests</li>
    <li>flatpak-builder.man</li>
    <li>flatpak.debug</li>
    <li>flatpak.dev</li>
    <li>flatpak.devdoc</li>
    <li>flatpak.doc</li>
    <li>flatpak.installedTests</li>
    <li>flatpak.man</li>
    <li>fractal</li>
    <li>gnome-builder</li>
    <li>gnome-builder.devdoc</li>
    <li>gnome-software</li>
    <li>kdePackages.discover</li>
    <li>kdePackages.discover.debug</li>
    <li>kdePackages.discover.dev</li>
    <li>kdePackages.discover.devtools</li>
    <li>kdePackages.flatpak-kcm</li>
    <li>kdePackages.flatpak-kcm.debug</li>
    <li>kdePackages.flatpak-kcm.dev</li>
    <li>kdePackages.flatpak-kcm.devtools</li>
    <li>libsForQt5.flatpak-kcm</li>
    <li>malcontent-ui</li>
    <li>malcontent-ui.dev</li>
    <li>malcontent-ui.lib</li>
    <li>monitor</li>
    <li>ostree</li>
    <li>ostree-full</li>
    <li>ostree-full.dev</li>
    <li>ostree-full.installedTests</li>
    <li>ostree-full.man</li>
    <li>ostree-rs-ext</li>
    <li>ostree.dev</li>
    <li>ostree.installedTests</li>
    <li>ostree.man</li>
    <li>pantheon.appcenter</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-onboarding</li>
    <li>pantheon.sideload</li>
    <li>pantheon.switchboard-plug-applications</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>rpm-ostree</li>
    <li>rpm-ostree.dev</li>
    <li>rpm-ostree.devdoc</li>
    <li>rpm-ostree.man</li>
    <li>streamcontroller</li>
    <li>vala-language-server</li>
    <li>xdg-desktop-portal</li>
    <li>xdg-desktop-portal-gnome</li>
    <li>xdg-desktop-portal-gtk</li>
    <li>xdg-desktop-portal-shana</li>
    <li>xdg-desktop-portal-xapp</li>
    <li>xdg-desktop-portal.installedTests</li>
  </ul>
</details>

---

###### Maintainer pings

cc @copumpkin for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

jtojnar committed:
ostree: Replace libsoup_2_4 with libsoup_3

We are using curl backend since 58ed9b3b2a387f1bfcfc9c1525e0c4c76fcc33aa
but libsoup is still used for trivial-httpd for tests.
Let’s just switch to libsoup_3 so that we can drop 2.4.

Also since we no longer use libsoup, we do not need glib-networking or
even wrapping (introduced in a350ad306aa8ee0a9d4fc28c533cad4e4c5493f5
to make libsoup support HTTPS).

jtojnar committed:
ostree.tests.installedTests: Fix after glib GIR files moved to glib

gobject-introspection 1.79 no longer ships GIR files for GLib since GLib 2.79 started generating them on its own:
https://gitlab.gnome.org/GNOME/gobject-introspection/-/merge_requests/433

That resulted in the following error:

    JS ERROR: Error: Requiring GLib, version 2.0: Typelib file for namespace 'GLib', version '2.0' not found

jtojnar committed:
ostree: Add ostree-full to passthru.tests

jtojnar commented:
Installed tests pass now:

```
machine # [  355.966410] gnome-desktop-testing-runner[743]: SUMMARY: total=136; passed=136; skipped=0; failed=0; user=52.1s; system=33.5s; maxrss=403884
```

jtojnar commented on pkgs/by-name/os/ostree/package.nix:
```diff
@@ -65,15 +64,19 @@ stdenv.mkDerivation rec {
-    url = "https://github.com/ostreedev/ostree/releases/download/v${version}/libostree-${version}.tar.xz";
-    sha256 = "sha256-VOM4fe4f8WAxoGeayitg2pCrf0omwhGCIzPH8jAAq+4=";
+    url = "https://github.com/ostreedev/ostree/releases/download/v${finalAttrs.version}/libostree-${finalAttrs.version}.tar.xz";
+    sha256 = "sha256-8kSkCMkJmYp3jhJ/zCLBtQK00BPxXyaUj0fMcv/i7vQ=";
```
Good catch, fixed.

jtojnar commented on pkgs/by-name/os/ostree/package.nix:
```diff
@@ -65,15 +64,19 @@ stdenv.mkDerivation rec {
   ];
 
   src = fetchurl {
```
In the absence of policy saying otherwise, I prefer to use the official distribution method.

LordGrimmauld commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 302813`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 2 packages marked as broken and skipped:</summary>
  <ul>
    <li>libsForQt5.discover</li>
    <li>plasma5Packages.discover</li>
  </ul>
</details>
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>bootc</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 64 packages built:</summary>
  <ul>
    <li>cosmic-store</li>
    <li>debos</li>
    <li>flatpak</li>
    <li>flatpak-builder</li>
    <li>flatpak-builder.doc</li>
    <li>flatpak-builder.installedTests</li>
    <li>flatpak-builder.man</li>
    <li>flatpak.debug</li>
    <li>flatpak.dev</li>
    <li>flatpak.devdoc</li>
    <li>flatpak.doc</li>
    <li>flatpak.man</li>
    <li>fractal</li>
    <li>gnome-builder</li>
    <li>gnome-builder.devdoc</li>
    <li>gnome-software</li>
    <li>haskellPackages.gi-ostree</li>
    <li>haskellPackages.gi-ostree.doc</li>
    <li>haskellPackages.ostree-pin</li>
    <li>haskellPackages.ostree-pin.doc</li>
    <li>kdePackages.discover</li>
    <li>kdePackages.discover.debug</li>
    <li>kdePackages.discover.dev</li>
    <li>kdePackages.discover.devtools</li>
    <li>kdePackages.flatpak-kcm</li>
    <li>kdePackages.flatpak-kcm.debug</li>
    <li>kdePackages.flatpak-kcm.dev</li>
    <li>kdePackages.flatpak-kcm.devtools</li>
    <li>libsForQt5.flatpak-kcm (plasma5Packages.flatpak-kcm)</li>
    <li>malcontent-ui</li>
    <li>malcontent-ui.dev</li>
    <li>malcontent-ui.lib</li>
    <li>monitor</li>
    <li>ostree</li>
    <li>ostree-full</li>
    <li>ostree-full.dev</li>
    <li>ostree-full.installedTests</li>
    <li>ostree-full.man</li>
    <li>ostree-rs-ext</li>
    <li>ostree.dev</li>
    <li>ostree.installedTests</li>
    <li>ostree.man</li>
    <li>pantheon.appcenter</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-onboarding</li>
    <li>pantheon.sideload</li>
    <li>pantheon.switchboard-plug-applications</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>rpm-ostree</li>
    <li>rpm-ostree.dev</li>
    <li>rpm-ostree.devdoc</li>
    <li>rpm-ostree.man</li>
    <li>streamcontroller</li>
    <li>vala-language-server</li>
    <li>wemeet</li>
    <li>xdg-desktop-portal</li>
    <li>xdg-desktop-portal-gnome</li>
    <li>xdg-desktop-portal-gtk</li>
    <li>xdg-desktop-portal-shana</li>
    <li>xdg-desktop-portal-termfilechooser</li>
    <li>xdg-desktop-portal-xapp</li>
    <li>xdg-desktop-portal.installedTests</li>
  </ul>
</details>

alyssais commented:
Checked musl/static stuff is still fine. :+1:
